app/helpers/application_helper.rb

```diff
@@ -211,13 +215,17 @@ def auto_complete_search(name, val, options = {})
   end
 
   def help_path
-    link_to _("Help"), :action => "welcome" if File.exist?("#{Rails.root}/app/views/#{controller_name}/welcome.html.erb")
+    link_to(_("Help"), { :action => "welcome" }, { :class => 'btn btn-default'}) if File.exist?("#{Rails.root}/app/views/#{controller_name}/welcome.html.erb")
   end
 
   def method_path(method)
     send("#{method}_#{controller_name}_path")
   end
 
+  def hash_method_path(method)
+    send("hash_for_#{method}_#{controller_name.singularize}_path")
```

Comment:
I'll get rid of this method completely. I don't even need the controller I believe, just the action.